### PR TITLE
ARROW-9581: [Dev][Release] Bump next snapshot versions to 2.0.0

### PR DESCRIPTION
**Pull request by kszucs**

### CODE REVIEW

File: r/NEWS.md [hunk 1]
```diff
@@ -19,6 +19,8 @@
 
 # arrow 1.0.0.9000
 
+# arrow 1.0.0.9000
+
```
Comment (kou) [inline r/NEWS.md:23]: Could you revert this?

**Review by kou — APPROVED** [review]
+1

**Comment by kou** [comment]
@github-actions crossbow submit -g linux

**Comment by kszucs** [comment]
+1, the build failures are unrelated.

**Pull request closed without merging**